docs/source/hardware/low_priority_faqs.md

```diff
@@ -0,0 +1,210 @@
+::::
+
+::::{dropdown} 2. What do I need to know about preemption and are there any drawbacks for my research group?
+
+**For private partition owners:** 
+When low priority jobs are running on your hardware, you may experience a slight delay (~30s) when submitting your first job to your private partition. This is the time it will take the scheduler to kill/preempt low priority jobs running on your partition. This new service is designed to minimize that delay.
+
+**For users of the** `lowpriority` **partition:** 
+Submitting jobs to this partition will make your jobs preemptable. That means they may be stopped/suspended and re-queued at any time. We recommend that you use strategies such as checkpointing (see {ref}`checkpoint-jobs`) to take full advantage of the `lowpriority` partition. RC is working on detailed documentation (see {ref}`low-priority`) and training sessions that will be available later in the Fall 2023 semester, once the service is ready to be released.
+::::
+
+::::{dropdown} 3. How does re-queuing of preempted (terminated) low priority jobs work?
+Preempted jobs are put back in the default partition queue (`short`) and scheduled normally.
+::::
+
+::::{dropdown} 4. My servers are being used by others, how fast can we retrieve them?
+As a private partition owner you will always have the highest priority when accessing your own resources. When submitting jobs to your private partition, any low priority jobs currently running on your servers will be killed and sent back to the queue. You will retrieve your server(s) within approximately 30s.
+::::
+
+::::{dropdown} 5. I already have access to one or more of the large, long, and/or multigpu partition(s), do I benefit from using the `lowpriority` partition?
+
+The goal of `lowpriority` partition is to double the resources available to Discovery users. Hence, specifying the `lowpriority` partition in your SLURM job header gives your job a higher chance of being allocated resources, even when your first choice of partition is unavailable. For e.g., when you specify `#SBATCH --partition=short,lowpriority`, your job can start running on the `lowpriority` partition even when `short` is unavailable. 
+::::
+
+:::{button-link} faq.html
+:color: primary
+:align: center
+Back to the top
+:::
+
+## RC Policy
+
+::::{dropdown} 1. Can I opt-out?
+We would prefer if you did not, but can if it's really required due to constraints like funding stipulations. Opening up idle private resources that consume power and cooling makes the cluster more useful for everyone (including you). This is a common practice in HPC implemented by many other universities and national labs. Also, all private hardware purchases that were placed since 2019 came with the explicit understanding that they would be added to the `lowpriority` partition when it became available (see next question).
```

Comment:
```suggestion
Opening up idle private resources that consume power and cooling makes the cluster more useful for everyone (including you). This is a common practice in HPC implemented by many other universities and national labs. Also, all private hardware purchases that were placed since 2019 came with the explicit understanding that they would be added to the `lowpriority` partition when it became available (see next question).

If your private resources have constraints (i.e., funding stipulations), please reach out to the RC team and we will remove your resources from the low priority partition.
```

Reply:
I made the change and pushed it. Thank you!!